src/components/Modal/ManageFundsModal.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import ModalModel from '../../models/modalModel';
+import ModalHeader from './ModalHeader/ModalHeader';
+import {
+  MarginedModalBody,
+  MobileAppLink,
+  ModalContentWrapper,
+  ModalWrapper,
+  OkButton
+} from './styles';
+
+const ManageFundsModal: React.SFC<{modal: ModalModel}> = ({modal}) => {
+  const handleCancel = (m: ModalModel) => () => {
+    m.close();
+  };
+
+  return (
+    <ModalWrapper>
+      <ModalHeader onClick={handleCancel(modal)} />
```

Comment:
you do not need to receive modal as param. It can be seen from the closure in the handleCancel